src/messages/messages_en.properties

```diff
@@ -23,8 +23,8 @@ missingPasswordMessage=Please specify password.
 missingEmailMessage=Please specify email.
 error-invalid-value=Invalid value.
 error-pattern-no-match=Invalid value.
-invalidPasswordMinLengthMessage=Invalid password: minimum length {0}.
-invalidPasswordConfirmMessage=Password confirmation doesn''t match.
+invalidPasswordMinLengthMessage=Invalid password\: minimum length {0}.
+invalidPasswordConfirmMessage=Password confirmation doesn't match.
```

Comment:
@teolemon is there a way to configure Crowdin to ensure that apostrophes are escaped properly?

Reply:
Don't worry, I found the setting for the crowdin.yml file